CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 0.4.0
 
-- Added support for add-to-app Flutter modules. Use `PfsTarget=` to set Fullstory version on Android if you encounter issues.
+- Added support for [add-to-app](https://docs.flutter.dev/add-to-app#add-to-ios-applications) Flutter modules. Use `PfsTarget=` to set Fullstory version on Android if you encounter issues.
```

Comment:
Good call, but perhaps it should go to https://docs.flutter.dev/add-to-app#add-to-android-applications or just https://docs.flutter.dev/add-to-app ?

Also, I think `PfsTarget=` should be `-PfsTarget=`, right?

Reply:
Good catch!